From 5c6a2d58d1e229a9c6aedafc6a1e1dfa13f5bacd Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Tue, 4 Mar 2025 18:18:11 +0100
Subject: [PATCH 817/954] refactor(stm32mp21): remove CCB register

CCB peripheral is no more supported.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 8b61e16c92d4be06f0ed9e08650dcf37431e5fab refactor(stm32mp21): remove CCB register]
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Change-Id: Id3acf5f205d1aeb55252dc7304e0c560af4cbfd0
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/441728
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
---
 include/drivers/st/stm32mp21_rcc.h | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/include/drivers/st/stm32mp21_rcc.h b/include/drivers/st/stm32mp21_rcc.h
index f3d24d53dd..6bcffd66c1 100644
--- a/include/drivers/st/stm32mp21_rcc.h
+++ b/include/drivers/st/stm32mp21_rcc.h
@@ -411,7 +411,6 @@
 #define RCC_HASH2CFGR				U(0x884)
 #define RCC_CRYP1CFGR				U(0x888)
 #define RCC_CRYP2CFGR				U(0x88C)
-#define RCC_CCBCFGR				U(0x890)
 #define RCC_IWDG1CFGR				U(0x894)
 #define RCC_IWDG2CFGR				U(0x898)
 #define RCC_IWDG3CFGR				U(0x89C)
@@ -3320,11 +3319,6 @@
 #define RCC_CRYPxCFGR_CRYPxEN			BIT(1)
 #define RCC_CRYPxCFGR_CRYPxLPEN			BIT(2)
 
-/* RCC_CCBCFGR register fields */
-#define RCC_CCBCFGR_CCBRST			BIT(0)
-#define RCC_CCBCFGR_CCBEN			BIT(1)
-#define RCC_CCBCFGR_CCBLPEN			BIT(2)
-
 /* RCC_IWDG1CFGR register fields */
 #define RCC_IWDG1CFGR_IWDG1EN			BIT(1)
 #define RCC_IWDG1CFGR_IWDG1LPEN			BIT(2)
-- 
2.53.0

